### PR TITLE
Testing the new yml file for extract only does one check.

### PR DESCRIPTION
**Pull request by mehrdadhalali**
I made the dashboard test fail and the `yml` file for `extract` check branches that only start with `extract-`, so it should pass if my syntax is correct.

### CODE REVIEW

File: .github/workflows/dashboard.yml [hunk 1]
```diff
@@ -2,9 +2,11 @@ name: Check Dashboard
 
 on:
   push:
-    branches: [main]
+    branches: 
+      - 'dashboard-*'
   pull_request:
-    branches: [main]
+    branches:
```
Comment (Leon-JS02) [inline .github/workflows/dashboard.yml:8]: This is a smart way of handling this.

File: dashboard/test_dashboard.py [hunk 2]
```diff
@@ -4,4 +4,4 @@
 
 
 def test_placeholder():
-    assert path.exists('README.md')
+    assert False
```
Comment (Leon-JS02) [inline dashboard/test_dashboard.py:7]: Should this be failing?

Comment (mehrdadhalali) [reply dashboard/test_dashboard.py:7]: Yes, I wanted to see if the overall workflow passes, given by branch name doesn't start with `dashboard-`.

Comment (Leon-JS02) [reply dashboard/test_dashboard.py:7]: Very well.

**Review by Leon-JS02 — DISMISSED** [review]
An efficient change. This will make sure unnecessary workflows aren't run.

File: dashboard/test_dashboard.py (hunk 2 above)
Comment (mehrdadhalali) [reply dashboard/test_dashboard.py:7]: Can do

Comment (MichaelHassan) [reply dashboard/test_dashboard.py:7]: otherwise the workflow wouldn't break, but it would cause a failing test in another part of the project

**Review by MichaelHassan — COMMENTED** [review]
This is great efficiency wise, but what happens if you change something elsewhere in the code. I.e Lets say a new feature requires changes in another part - the load script may need changes in the extract scripts. Any issues wouldn't be caught straight away.

**Review by Leon-JS02 — APPROVED** [review]
Re-approval. Made a failed placeholder test pass - still valid.

**Review by paulauus — APPROVED** [review]
Happy if all tests pass. Efficient update - learnt something new about yml.

**Pull request merged**